### DISCUSSION TITLE
Entry ClearButton does not clear the text?

### DISCUSSION THREAD

**Discussion by FreakyAli** [dotnet/maui#9919] [General] [answered]
So earlier today I was trying to use the internal property for an Entry to clear the text in it.

     <Entry
       Placeholder="This is a freaky entry"
       TextColor="Purple"
       ClearButtonVisibility="WhileEditing"
       FontSize="Large"
       Keyboard="Chat"/>

But when I click on it nothing seems to happen. 

Am I missing something here?

Is this the intended behaviour? 

If so how do I add the clear button functionality, I don't seem to find a command or a event for it...

**Comment by hartez** [comment, marked as answer]
That looks like it should be working. I'd suggest opening an issue.

**Reply by FreakyAli** [reply]
Are you able to reproduce this too?
